CHANGES.md

```diff
@@ -0,0 +1,11 @@
+# Version 0.4.0
+
+* Several enumerations are now non_exhaustive for future extensions.
+  These are `Tag`, `Type`, `Value`, `PhotometricInterpretation`,
+  `CompressionMethod`, `Predictor`.
+* Enums gained a dedicated method to convert to their TIFF variant value with
+  the specified type. Performing these conversions by casting the discriminant
+  with `as` is not guaranteed to be stable, except where documented explicitly.
+* Removed the num-derive and num dependencies.
+* Added support for compressed images.
```

Comment:
Something like "Added support for decoding images compressed with the deflate method" would be more accurate here I think: we already supported some compression methods before that, at least for decoding (LZW and PackBits it looks like), what was added is deflate support in #49, and only for decoding apparently.